artiq/coredevice/phaser.py

```diff
@@ -1039,6 +1057,44 @@ def en_trf_out(self, rf=1, lo=0):
             data = data ^ ((1 << 12) | (1 << 13))
         self.trf_write(data)
 
+    @kernel
+    def set_servo(self, bypass=1, hold=0, profile=0):
+        """Set the servo configuration.
+
+        :param bypass: 1 to enable bypass (default), 0 to engage servo
+        :param hold: 1 to hold the servo IIR filter output constant, 0 for normal operation
+        :param profile: profile index to select for channel (0 to 3)
+        """
+        if (profile < 0) | (profile > 3):
+            raise ValueError("invalid profile index")
+        addr = PHASER_ADDR_SERVO_CFG1 if self.index == 1 else PHASER_ADDR_SERVO_CFG0
+        if bypass == 0:
+            data = 1
+        if hold == 1:
+            data = data | (1 << 1)
+        data = data | (profile << 2)
+        self.phaser.write8(addr, data)
+
+    @kernel
+    def set_iir_mu(self, profile, ab, offset):
```

Comment:
Is it `ab` or `ba`?
I wouldn't pass a list. Seems unnecessary here and doesn't help much.